macosx/QuickLookExtension/PreviewProvider.h

```diff
@@ -0,0 +1,10 @@
+// This file Copyright © Transmission authors and contributors.
+// It may be used under the MIT (SPDX: MIT) license.
+// License text can be found in the licenses/ folder.
+
+#import <Cocoa/Cocoa.h>
+#import <Quartz/Quartz.h>
```

Comment:
I don't think we need to import Cocoa or Quartz, just QuickLookUI should be enough?

Reply:
This is default imports when you create QuickLook Preview extension with Xcode.
If it works - it works. I don't think this needs to be updated for now.